### ISSUE TITLE
Compare with cargo-license and show some example output in README.

### ISSUE THREAD

**Issue by vi** [Nemo157/cargo-lichking#42]
Is it the same as [cargo-license](https://github.com/onur/cargo-license)? In any case, README should be extented.

**Comment by Nemo157** [comment]
I'm not sure whether I want a comparison against cargo-license, but definitely should enhance the readme, lots of new features to highlight.

**Comment by arve0** [comment]
I'm expecting to see some output on check, like "OK", but only get the IANAL warning. Is this correct behavior?

```
$ cargo lichking check
warning: IANAL: This is not legal advice and is not guaranteed to be correct.
$ echo $?
0
```

**Comment by Nemo157** [comment]
@arve0 that's correct currently, but I think it makes sense to print something saying it was fine, opened #53 for it.
